app/models/version.rb

```diff
@@ -38,7 +38,6 @@ class Version < ApplicationRecord
   ].freeze
 
   validates :project_id, :number, presence: true
-  validates :number, uniqueness: { scope: :project_id }
```

Comment:
I think this validation had come up in a PR earlier, which I can't find now, but this should be fine since there's a UNIQUE index on these columns already